develop/blockchains/get-started/intro-polkadot-sdk.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Introduction to Polkadot SDK
+description: Learn about the Polkadot SDK, what it is comprised of, and how it powers and forms the foundation of the Polkadot protocol
+---
+
+# Introduction to Polkadot SDK
+
+## Introduction
+
+The [Polkadot SDK](https://github.com/paritytech/polkadot-sdk){target=\_blank} is a powerful and versatile developer kit designed to facilitate building on the Polkadot network. It provides the necessary components for creating custom blockchains, parachains, generalized rollups, and more. Written in the Rust Programming Language, it puts security and robustness at the forefront of its design.
+
+Whether you're building a standalone chain or deploying a parachain on Polkadot, this SDK equips developers with the libraries and tools needed to manage runtime logic, compile the codebase, and utilize core features like staking, governance, and Cross-Consensus Messaging (XCM). It also provides a means for building generalized peer-to-peer systems, whether a blockchain or an alternative technology. The Polkadot SDK houses the following overall functionality:
+
+- Networking and peer-to-peer communication (powered by [Libp2p](/polkadot-protocol/glossary#libp2p))
+- Consensus protocols, such as [BABE](/polkadot-protocol/glossary#blind-assignment-of-blockchain-extension-babe), [GRANDPA](/polkadot-protocol/glossary#grandpa), or [Aura](/polkadot-protocol/glossary#authority-round-aura)
+- Cryptography
+- A selection of pre-built modules, called [pallets](/polkadot-protocol/glossary#pallet)
+- Benchmarking and testing suites
+- The ability to create portable Wasm runtimes
+
+!!!note
+    For an in-depth dive into the monorepo, the [Polkadot SDK Rust documentation](https://paritytech.github.io/polkadot-sdk/master/polkadot_sdk_docs/polkadot_sdk/index.html){target=\_blank} is highly recommended.
+
+## Polkadot SDK Overview
+
+The Polkadot SDK is composed of five major components:
+
+![](/images/develop/blockchains/get-started/intro-polkadot-sdk/polkadot-sdk-structure.webp)
```

Comment:
If we replace this image with a diagram created by us, I'd highly recommend encapsulating the five components into the Polkadot SDK.